verification/curator-service/ui/package.json

```diff
@@ -26,11 +26,14 @@
   "scripts": {
     "start-noenv": "react-scripts start",
     "start": "PORT=3002 react-scripts start",
+    "start-full-stack": "./../../../dev/run_stack.sh",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "test-silent": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
-    "cypress": "cypress open"
+    "cypress-open": "cypress open",
+    "cypress-run": "cypress run",
+    "cypress-ci-test": "WAIT_ON_TIMEOUT=600000 start-server-and-test start-full-stack http://localhost:3002 cypress-run"
```

Comment:
building and starting the stack should be a github actions step, cf how the current docker workflow work:
https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/.github/workflows/docker.yml#L18

once you've done that then you can npm run cypress-ci-test that points to localhost:3002, no need for start-server-and-test and start-full-stack scripts anymore

Reply:
Ahh thanks! Done.